cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,288 @@
+
+// TODO: Move the theme variables to cf-enhancements.
+/* topdoc
+  name: Theme variables
+  family: cf-core
+  notes:
+    - "The following color and sizing variables are exposed, allowing you to
+       easily override them before compiling."
+  patterns:
+    - name: Sizing
+      codenotes:
+        - |
+          @margin__em
+          @margin_half__em
+  tags:
+    - cf-core
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+@margin_half__em: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```

Comment:
Maybe the name should be something like `@base-space__em`, `@base-space_half__em`? That way it would also not imply it's for use only in a `margin` style.

@sebworks what do you think?